pkg/ruptela/schema/oids.csv

```diff
@@ -1667,7 +1667,7 @@ Range: 0~250, Offset: 0",No,Yes,No,No,Yes,Yes,Yes,Yes,No,No,No,No,No,No,No,No,No
 961,OBD tire pressure front right,2,Unsigned int.,0,65534,"0,05; 0",PSI,65535,,Yes,No,Yes,No,Yes,Yes,No,Yes,Yes,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No
 962,OBD tire pressure rear left,2,Unsigned int.,0,65534,"0,05; 0",PSI,65535,,Yes,No,Yes,No,Yes,Yes,No,Yes,Yes,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No
 963,OBD tire pressure rear righ,2,Unsigned int.,0,65534,"0,05; 0",PSI,65535,,Yes,No,Yes,No,Yes,Yes,No,Yes,Yes,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No
-964,OBD Oil Life remaining,1,Unsigned int.,0,254,"0,393;0",%,255,,No,No,Yes,No,Yes,Yes,No,Yes,Yes,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No
+964,OBD Oil Life remaining,1,Unsigned int.,0,254,"1;0",%,255,,No,No,Yes,No,Yes,Yes,No,Yes,Yes,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No,No

```

Comment:
here I changed oil multiplier

Reply:
was this a manual edit or did you pull the latest FMIOList

Reply:
we pulled the latest FMIOList